spiceaidocs/content/en/concepts/_index.md

```diff
@@ -46,11 +46,15 @@ Spice.ai provides a general interface that anyone can implement to create a data
 
 [Actions]({{<ref "reference/pod#actions">}}) are the set of actions the Spice.ai runtime can recommend for a pod.
 
+### Recommendations
+
+After a pod has been trained, it will return intelligent [recommendations]({{<ref "concepts/recommendations">}}) which identify an action to take at a particular time. By default a recommendation will use the latest data available to recommend an action.
```

Comment:
```suggestion
After a pod has been trained, it will return intelligent [recommendations]({{<ref "concepts/recommendations">}}) which identifies an action to take at a particular time. By default, a recommendation will use the last ingested observation to recommend an action.
```